src/linear_algebra/char_poly/coeff.lean

```diff
@@ -4,13 +4,14 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Aaron Anderson, Jalex Stark
 -/
 
+import algebra.polynomial.big_operators
 import data.matrix.char_p
+import field_theory.finite.basic
+import group_theory.perm.cycles
 import linear_algebra.char_poly.basic
 import linear_algebra.matrix
 import ring_theory.polynomial.basic
-import algebra.polynomial.big_operators
-import group_theory.perm.cycles
-import field_theory.finite.basic
+import ring_theory.power_basis
```

Comment:
I took the opportunity to sort the imports. The net change consists of adding `ring_theory.power_basis`.